.github/workflows/clippy.yml

```diff
@@ -0,0 +1,83 @@
+name: Clippy Test
+
+on:
+  push:
+    # Ignore bors branches, since they are covered by `clippy_bors.yml`
+    branches-ignore: [auto, try]
+    # Don't run Clippy tests, when only textfiles were modified
+    paths-ignore:
+    - 'COPYRIGHT'
+    - 'LICENSE-*'
+    - '**.md'
+    - '**.txt'
+  pull_request:
+    # Don't run Clippy tests, when only textfiles were modified
+    paths-ignore:
+    - 'COPYRIGHT'
+    - 'LICENSE-*'
+    - '**.md'
+    - '**.txt'
+
+env:
+  RUST_BACKTRACE: 1
+  CARGO_TARGET_DIR: '${{ github.workspace }}/target'
+  GHA_CI: 1
+
+jobs:
+  base:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: rust-lang/simpleinfra/github-actions/cancel-outdated-builds@master
+      with:
+        github_token: "${{ secrets.github_token }}"
+    - name: rust-toolchain
+      uses: actions-rs/toolchain@v1.0.3
+      with:
+        toolchain: nightly
+        target: x86_64-unknown-linux-gnu
+        profile: minimal
+    - name: Checkout
+      uses: actions/checkout@v2.0.0
+    - name: Run cargo update
+      run: cargo update
+    - name: Cache cargo dir
+      uses: actions/cache@v1
+      with:
+        path: ~/.cargo
+        key: ${{ runner.os }}-x86_64-unknown-linux-gnu-${{ hashFiles('Cargo.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-x86_64-unknown-linux-gnu
+    - name: Master Toolchain Setup
+      run: bash setup-toolchain.sh
+
+    - name: Set LD_LIBRARY_PATH (Linux)
+      run: |
+        SYSROOT=$(rustc --print sysroot)
+        echo "::set-env name=LD_LIBRARY_PATH::${SYSROOT}/lib${LD_LIBRARY_PATH+:${LD_LIBRARY_PATH}}"
```

Comment:
Don't want to start a formatting bikeshed, but to me yaml steps always read better if they are separated by an empty line each. For comparison:

```yaml
    - name: Build
      run: cargo build --features deny-warnings
    - name: Test
      run: cargo test --features deny-warnings
    - name: Test clippy_lints
      run: cargo test --features deny-warnings
      working-directory: clippy_lints
    - name: Test rustc_tools_util
      run: cargo test --features deny-warnings
      working-directory: rustc_tools_util
```

```yaml
    - name: Build
      run: cargo build --features deny-warnings

    - name: Test
      run: cargo test --features deny-warnings

    - name: Test clippy_lints
      run: cargo test --features deny-warnings
      working-directory: clippy_lints

    - name: Test rustc_tools_util
      run: cargo test --features deny-warnings
      working-directory: rustc_tools_util
```

Reply:
Hm I grouped it into "Setup"-steps, "Run"-steps, and "post-run"-steps. But I can format it like this and just write `# Setup`, `# Run`, ... above these steps. I'll change it 👍